asApp/src/asVerify.c

```diff
@@ -52,23 +57,24 @@ int main(int argc,char **argv)
 	int		numDifferences;
 	int		status;
 	int		verbose=0, debug=0, write_restore_file=0;
+	int		opt;
+
+	while ((opt = getopt(argc, argv, "Vvrdh")) != -1) {
```

Comment:
getopt is POSIX only, is it not? So this wouldn't be able to build under windows?

Reply:
Good catch. I have switched it to use `epicsGetopt` which should be Windows compliant as well.